sensor/core/models.py

```diff
@@ -0,0 +1,47 @@
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.db import models
+
+
+class GenericSensor(models.Model):
+    """Represents a sensor abstracting away the specifics of what it measures.
+
+    A sensor measures one kind of thing. A physical device might have
+    multiple logical sensors.
+    """
+
+    name = models.CharField(max_length=256)
+    model = models.CharField(max_length=128)
```

Comment:
Pole model jest w kontekście Django trochę wieloznaczne - jeżeli ma to być pole przechowujące nazwę modelu sensora (typu np. 'dr12-23c' to sugerowałbym jakoś inaczej to pole nazwać